gateways/class-wc-ebanx-bank-transfer-gateway.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+use Ebanx\Benjamin\Models\Country;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WC_EBANX_Bank_Transfer_Gateway
+ */
+class WC_EBANX_Bank_Transfer_Gateway extends WC_EBANX_New_Gateway {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->id           = 'ebanx-banktransfer';
+		$this->method_title = __( 'Transferência Bancária', 'woocommerce-gateway-ebanx' );
+
+		$this->api_name = 'banktransfer';
+
+		$this->title = 'Transferência Bancária';
+
+		$this->description = 'Pague com transferência bancária usando o saldo da sua conta corrente. Importante: Depósitos em dinheiro não serão aceitos.';
+
+		parent::__construct();
```

Comment:
It`s not better to move this line to line 18?

Reply:
If I move up there the icon breaks, some other thing may break as well, I copied the call order of the other gateways.